index.html

```diff
@@ -5,8 +5,31 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Sam - Portofolio</title>
+    <script src="https://kit.fontawesome.com/b39c945343.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="assets/style.css" />
 </head>
 <body>
-    
+    <header id="header_menu">
+        <a class="welcome_text">
+            Welcome
+        </a>
+        <a href="#" class="toggleNav">
+            <img src="assets/icons/union.png" alt="union icon" />
+        </a>
+    </header>
+    <section id="main_section">
+        <h1 id="presentation_text">
+            <span class="greetings">Hey there. I’m Sammy</span> <br>
+            <span class="profession">I’m a software developer</span></h1>
+        <p id="biography">
+             I can help you build a dream product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to reach out to me.
+        </p>
+        <div id="social_media_links">
+            <a href="https://github.com/ndaboom" target="_blank"><i class="fa-brands fa-github fa-2xl"></i></a>
+            <a href="https://www.linkedin.com/in/sam-ndabo-b0431b17b/" target="_blank"><i class="fa-brands fa-linkedin fa-2xl"></i></a>
+            <a href="https://twitter.com/ndabosammy" target="_blank"><i class="fa-brands fa-twitter fa-2xl"></i></a>
+            <a href="https://www.facebook.com/sammy.ndabo/" target="_blank"><i class="fa-brands fa-facebook fa-2xl"></i></a>
+        </div>
```

Comment:
- [ ] Kindly replace this `div` element with a `ul` element. Each icon should be inside `a` tag wrapped in `li` 💯